scene/2d/gpu_particles_2d.h

```diff
@@ -143,3 +154,5 @@ class GPUParticles2D : public Node2D {
 VARIANT_ENUM_CAST(GPUParticles2D::DrawOrder)
 
 #endif // PARTICLES_2D_H
+
+void _calc_vis_rect_hvframes(CanvasItemMaterial *p_material);
```

Comment:
You already have it defined in the class, and now you're defining it outside header guards so it will be included in all compilation units.

Reply:
I don't know how this went there...